src/files/file-icon/FileIcon.js

```diff
@@ -11,21 +11,21 @@ import DocText from '../../icons/GlyphDocText'
 
 export default function FileIcon ({name, type}) {
   if (type === 'directory') {
-    return <Folder className=' fill-aqua' width='2.5rem' />
+    return <Folder className=' fill-aqua' style={{width: '2.5rem'}} />
```

Comment:
The `style={{width: '2.5rem'}}` should be in a variable as it's being used in all the cases.